### DISCUSSION TITLE
Home Assistant OS 14.1

### DISCUSSION THREAD

**Discussion by sairon** [home-assistant/operating-system#3753] [Announcements]
## Home Assistant Operating System

* Bump Buildroot to 2024.02.9 (#3745) @sairon
* Enable TCPMSS target iptables module for all boards (#3739) @sairon

## Raspberry Pi

* RaspberryPi: Update kernel to 6.6.62 - dd2394360860d15146c96635796a75b05bb32b61 (#3715) @sairon
* Compile hailo_pci module from original sources on RPi 5 (#3732) @sairon
* Fix Hailo-8 firmware path expected since driver v4.19.0 (#3721) @sairon
* Bump RPi SDIO WiFi firmware to latest version (#3719) @sairon
* Bump Hailo-8 firmware to 4.19.0 for RPi kernel 6.6.62 (#3716) @sairon

## Home Assistant Yellow

* RaspberryPi: Update kernel to 6.6.62 - dd2394360860d15146c96635796a75b05bb32b61 (#3715) @sairon
* Remove existing rauc.db from a data disk on the first boot (#3737) @sairon
* Add Hailo PCIe driver and firmware for Yellow (#3734) @sairon
* Bump RPi SDIO WiFi firmware to latest version (#3719) @sairon

## Home Assistant Green

* Linux: Update kernel to 6.6.66 (#3744) @sairon

## Open Virtual Appliance

* Linux: Update kernel to 6.6.66 (#3744) @sairon
* Add Hailo PCIe driver and firmware for x86-64, aarch64 and OVA (#3736) @sairon

## Generic x86-64

* Linux: Update kernel to 6.6.66 (#3744) @sairon
* Add Hailo PCIe driver and firmware for x86-64, aarch64 and OVA (#3736) @sairon

## Hardkernel ODROID

* Linux: Update kernel to 6.6.66 (#3744) @sairon

## ASUS Tinker

* Linux: Update kernel to 6.6.66 (#3744) @sairon

## Khadas VIM Series

* Linux: Update kernel to 6.6.66 (#3744) @sairon

## Generic aarch64

* Linux: Update kernel to 6.6.66 (#3744) @sairon
* Add Hailo PCIe driver and firmware for x86-64, aarch64 and OVA (#3736) @sairon

## Build System

* Add HA Yellow image to RPi Imager index update action (#3731) @sairon
* Bump docker/setup-buildx-action from 3.7.1 to 3.8.0 (#3746) @dependabot

<hr /><em>This discussion was created from the release <a href='https://github.com/home-assistant/operating-system/releases/tag/14.1'>Home Assistant OS 14.1</a>.</em>

**Comment by Anto79-ops** [comment]
There are no issues on my NUC 12 (x86-64, bare-metal) with USB SkyConnect and USB Aeotec Z-Stick 7.

The system is running normally.

**Comment by saschachina** [comment]
No issues on my ESPRIMO Q920 (Generic x86-64) with USB Sonoff Dongle P (CC2652)

**Comment by Glodarilie** [comment]
if i make update i cant connect anymore to my HA, i have a raspberry pi5, if i unplug and plug back, it come back but update is not done

**Reply by kaovilai** [reply]
[Same issue I think](https://github.com/home-assistant/operating-system/discussions/3753#discussioncomment-11641272). So this means I need an independent from HA, Wi-Fi internet connected plug to force restart Pi5 to avoid HA outage like this.

**Comment by kaovilai** [comment]
After clicking update from Web UI (from another country conected via [Tailscale Funnel](https://github.com/hassio-addons/addon-tailscale/blob/main/tailscale/DOCS.md#option-funnel)), the RPi5 restarted and I am unable to reconnect to the device via the Funnel or other means.

Remotely checking the AT&T Nokia Router/Gateway(BGW320-505) via Smart Home Manager app reveals the device is no longer ethernet connected. Will get back to the device in few weeks to investigate, but for now my house is only smart for internet connected devices.

**Comment by Oktandrew** [comment]
Was unable to update from 14.0 to 14.1 on a Raspberry Pi 5.

The system was stuck or otherwise hanged and required a manual reboot.
After rebooting the device was still on 14.0 and works properly.

**Comment by Glodarilie** [comment]
I had same problem but I retried after a day and worked fine after,now all
good and have latest os

**Comment by 2810159288** [comment]
Downloaded several times, it says the image is corrupted   X86

**Comment by WowIMadeThis** [comment]
I'm having the same issue with my installation on Raspberry Pi 5. 

Updating to 14.1 fails, and I have to pull the power plug to restart it. It's working fine after restart, but it's still on 14.0.

I also tried to update by running "ha os update" through SSH, and it seemed to finish successfully, but the system failed in the same manner as before and after restart it was still on 14.0.

**Comment by ardtieboy** [comment]
Same here; update from 14.0 to 14.1 in the UI. After the system says its rebooting, its just not starting anymore. I need to pull the plug in order for it to restarted, however it stays in 14.0

**Comment by JohnMurphy474** [comment]
Same issue here with Raspberry Pi 5 not updating. Will not restart after the update. A manual physical restart works, but update not applied, still on 14.0. Can't see anything in logs indicating the problem. I have tried to update 3 times now with the same result.

**Comment by lion7** [comment]
Same problem here, also on a Raspberry pi 5.
I noticed @sairon is mentioned in a change regarding the Hailo PCIe Driver. I'm booting my pi from a NVMe SSD,  so it might be related.

**Reply by JohnMurphy474** [reply]
My boot is from SSD, no NVMe. Don't think that's it.

**Comment by JohnMurphy474** [comment]
I have eventually managed to get my Pi 5 updated to 14.1. I followed some advice from **Serl**  to reboot from Developer Tools > Restart > Advanced options > Reboot system. After re-boot completed I retried the update and am pleased to say that it restarted properly and I'm now running 14.1. I can't explain why all the other re-boots I had already done did not have the same effect.

**Comment by Peter376** [comment]
Is it safe to jump directly from OS 11.5 to 14.1?

Noob here. sorry guys. I am still on:
- OS version 11.5
- core 2024.3.1. 
HA on Pi4B with Skyconnect.

Would really appreciate your thoughts. 

Pete

**Reply by thezfunk** [reply]
I am in the same situation.  The Z-Wave dongle boot loop issue is why I am still on 11.5.  I have been hanging here until I get a confirmation that issue has been fixed.

**Comment by Nico128** [comment]
Same problem on PI 5 with a new installation via Rasberry Pi Image. Unfortunately I don't have any microHDMI cable so I cannot see the output. In the journal log I could see "success=no exit=-13 a0=ffffffffffffff9c"
The network interface got an IP but the web interface doesn't work (port 8123 closed)

I use a SD card.

I downloaded 14.0 and install it with the same tools and now it's working.

**Comment by RapscallionPearl** [comment]
I have had the same issue, with my Hass not doing anything after losing all connection, returning to alive after disconnecting power for a hard reset, 
I tried what https://github.com/home-assistant/operating-system/discussions/3753#discussioncomment-11748024 did but to no avail.
then set a screen to see the output of the Raspberry Pi 5 when rebooting, it just stops and does nothing, There is no update on the feed on the screen, but for some reason I did get a response when I plugged a keyboard in, to check if it was just external that was shut down. the error message was something about not being able to resolve Bluetooth, several instances of that, I did not manage to get a screen dump of it since it just rebooted shortly after, as if the power had been removed.. 

Then I shifted the keyboard to a different USB port to make sure that was not the case.
and in the boot, all looked normal, but still 14.0 
I rebooted in safe mode from the CLI and then told it to update
and kept peripherals in so I could monitor, but nothing special happened, and it booted up as it was supposed to after an update.

But now it booted up to 14.1 as intended.. 
I ran another reboot to confirm, and seems that did it, will post again should it drop out, but so far it is running as intended
